api/v1alpha1/ext_auth_types.go

```diff
@@ -33,6 +33,8 @@ type ExtAuth struct {
 	// +optional
 	HeadersToExtAuth []string `json:"headersToExtAuth,omitempty"`
 
+	BodyToExtAuth *string `json:"bodyToExtAuth,omitempty"`
```

Comment:
why this a string type and not a bool

Reply:
I don't know how it works so I don't know if it will receive a string which is the body of the request or a bool to activate it because `HeadersToExtAuth` is an array of string

Reply:
https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/filters/http/ext_authz/v3/ext_authz.proto#envoy-v3-api-msg-extensions-filters-http-ext-authz-v3-buffersettings is the underlying API in envoy

do we need to expose `max_request_bytes` to the user ?

Reply:
I don't think that's necessary for the moment

Reply:
In my opinion, all configuration options for with_request_body should be exposed. Otherwise, we risk facing the same issues that this PR is intended to solve, namely the lack of control over Envoy Proxy from the gateway. For instance, in the case of max_request_bytes, the Envoy documentation specifies that if the body exceeds this limit, Envoy will return HTTP 413 and will not initiate the authorization process once the buffer reaches the set value. This behavior can be a blocker, especially if this parameter isn’t configurable via the gateway.

Thank you very much for the great work!